build_defs/python.build_defs

```diff
@@ -588,7 +588,7 @@ def python_wheel(name:str, version:str, labels:list=[], hashes:list=None, packag
         # generating a url with the wheel_resolver tool and download that if successful.
         cmd = f'$TOOL --package {package_name} --version {version} --prereleases {prereleases}'
         if urls:
-            cmd += ' --urls ' + ' '.join(['%s' % url for url in urls])
+            cmd += ''.join([' --urls %s' % url for url in urls])
```

Comment:
Shouldn't this be `' '.join(f'--urls {url}' for url in urls])`?

Reply:
That also works. The above does because `--urls` is surrounded by spaces before and after the flag